internal/events/websockets/websocket_connection.go

```diff
@@ -88,9 +88,11 @@ func (wc *websocketConnection) processAutoStart(req *http.Request) {
 			Name:      query.Get("name"),
 			Filter: fftypes.SubscriptionFilter{
 				Events: query.Get("filter.events"),
-				Topics: query.Get("filter.topics"),
-				Group:  query.Get("filter.group"),
-				Tag:    query.Get("filter.tag"),
+				Message: fftypes.MessageFilter{
```

Comment:
Think we need to handle the old options too, to avoid breaking apps.

Also the docs have examples that use these, which should use the new syntax.